FreeRTOS-Plus/Source/Application-Protocols/network_transport/freertos_plus_tcp/using_wolfSSL/using_wolfSSL.c

```diff
@@ -382,21 +394,22 @@ TlsTransportStatus_t TLS_FreeRTOS_Connect( NetworkContext_t * pNetworkContext,
 
 void TLS_FreeRTOS_Disconnect( NetworkContext_t * pNetworkContext )
 {
-    WOLFSSL* pSsl = pNetworkContext->sslContext.ssl;
+    WOLFSSL * pSsl = pNetworkContext->sslContext.ssl;
 
     /* shutdown an active TLS connection */
-    wolfSSL_shutdown(pSsl);
+    wolfSSL_shutdown( pSsl );
 
     /* cleanup WOLFSSL object */
-    wolfSSL_free(pSsl);
+    wolfSSL_free( pSsl );
     pNetworkContext->sslContext.ssl = NULL;
 
     /* Call socket shutdown function to close connection. */
-    Sockets_Disconnect(pNetworkContext->tcpSocket);
+    Sockets_Disconnect( pNetworkContext->tcpSocket );
 
     /* free WOLFSSL_CTX object*/
-    WOLFSSL_CTX* pCtx = pNetworkContext->sslContext.ctx;
-    wolfSSL_CTX_free(pCtx);
+    WOLFSSL_CTX * pCtx = pNetworkContext->sslContext.ctx;
```

Comment:
(Nit) To be consistent with our coding style, the `pCtx` declaration can be at the top of the function body.